### DISCUSSION TITLE
Blender 3.4 crashing when enabling add-on v0.6.2

### DISCUSSION THREAD

**Discussion by subaruparts** [eliemichel/MapsModelsImporter#261] [Q&A]
Hi, I'm having trouble getting this add-on installed and functional in Blender 3.4. I've installed v0.6.2 in Blender using Edit > Preferences > Add-ons and then selecting the "MapsModelsImporter-v0.6.2.zip" downloaded from the release page. "Import-Export: Maps Model Importer" shows up in the Add-on list however when I try to enable the add-on, Blender crashes. I've tried v0.6.1 and v0.6.0 and have the same issue. Any ideas on what may be causing this? Thanks!
![Screenshot 2023-02-17 152300](https://user-images.githubusercontent.com/125703797/219815657-6f233103-0a89-4dcd-a2a7-dfc7a94d1bea.png)

**Comment by eliemichel** [comment]
Can you run Blender from a cmd so that you keep the log of console messages after Blender crashed, and report it here?

**Reply by subaruparts** [reply]
> Can you run Blender from a cmd so that you keep the log of console messages after Blender crashed, and report it here?

Sure, here is the log:
![Screenshot 2023-02-19 150038_ps](https://user-images.githubusercontent.com/125703797/219980778-06db082d-8b67-4f5f-8c3f-8b0039458e0f.png)

**Comment by brumemp5** [comment]
Hi, I have exactly the same problem as you, I managed to install the add on, import my rdc file, then suddenly it crashed, then I reopened blender and my add on disappeared then impossible to put it back in blender
![Capture d'écran_20230219_015316](https://user-images.githubusercontent.com/115560820/219906216-b46028ce-df3c-47e0-9cdf-2feb63f3e4d9.png)

**Comment by subaruparts** [comment]
This has since resolved itself. I can only assume a background update inadvertently fixed what may have been causing the issue.

I've been using the add on for a couple of weeks without issue.
